graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/types/generateDirective.kt

```diff
@@ -57,7 +57,9 @@ private fun getDirective(generator: SchemaGenerator, directiveInfo: DirectiveInf
         }
 
         val directiveClass = directiveInfo.directive.annotationClass
-        directiveClass.getValidProperties(generator.config.hooks).forEach { prop ->
+        val directiveArguments = directiveClass.getValidProperties(generator.config.hooks)
```

Comment:
whats the point of creating separate value? I guess this is a leftover from the refactor?

Reply:
Yea, I was using this to inspect the value and I was trying before the merge separate values together.